source/nodejs/adaptivecards-designer/src/card-designer.ts

```diff
@@ -1560,28 +1560,28 @@ export class ImagePeer extends TypedCardElementPeer<Adaptive.Image> {
             backgroundColor.input.defaultValue = this.cardElement.backgroundColor;
             backgroundColor.input.onValueChanged = () => {
                 this.cardElement.backgroundColor = backgroundColor.input.value;
-                
+
                 this.changed(false);
             }
-        }
 
-        var actionSelector = createActionSelector(card, this.cardElement.selectAction ? this.cardElement.selectAction.getJsonTypeName() : "none");
+            var actionSelector = createActionSelector(card, this.cardElement.selectAction ? this.cardElement.selectAction.getJsonTypeName() : "none");
 
-        actionSelector.input.onValueChanged = () => {
-            if (actionSelector.input.value == "none") {
-                this.cardElement.selectAction = null;
-            }
-            else {
-                this.cardElement.selectAction = Adaptive.AdaptiveCard.actionTypeRegistry.createInstance(actionSelector.input.value);
-            }
+            actionSelector.input.onValueChanged = () => {
+                if (actionSelector.input.value == "none") {
+                    this.cardElement.selectAction = null;
+                }
+                else {
+                    this.cardElement.selectAction = Adaptive.AdaptiveCard.actionTypeRegistry.createInstance(actionSelector.input.value);
+                }
 
-            this.changed(true);
-        }
+                this.changed(true);
+            }
```

Comment:
There seems to be an indentation problem with the lines above until this line. Seems too indented now.

Reply:
P2: nit